bindings/rust/bench/src/lib.rs

```diff
@@ -6,28 +6,60 @@ pub mod openssl;
 pub mod rustls;
 pub mod s2n_tls;
 pub use crate::{
-    harness::{CipherSuite, CryptoConfig, ECGroup, HandshakeType, Mode, TlsBenchHarness},
+    harness::{CipherSuite, CryptoConfig, ECGroup, HandshakeType, Mode, SigType, TlsBenchHarness},
     openssl::OpenSslHarness,
     rustls::RustlsHarness,
     s2n_tls::S2NHarness,
 };
 
-const SERVER_KEY_PATH: &str = "certs/server-key.pem";
-const SERVER_CERT_CHAIN_PATH: &str = "certs/server-fullchain.pem";
-const CLIENT_KEY_PATH: &str = "certs/client-key.pem";
-const CLIENT_CERT_CHAIN_PATH: &str = "certs/client-fullchain.pem";
-const CA_CERT_PATH: &str = "certs/ca-cert.pem";
+#[derive(Clone, Copy)]
+pub enum PemType {
+    ServerKey,
+    ServerCertChain,
+    ClientKey,
+    ClientCertChain,
+    CACert,
+}
+
+use PemType::*;
+use SigType::*;
+
+fn get_cert_path(pem_type: PemType, sig_type: SigType) -> String {
+    let filename = match pem_type {
+        ServerKey => "server-key.pem",
+        ServerCertChain => "server-cert.pem",
+        ClientKey => "client-key.pem",
+        ClientCertChain => "client-cert.pem",
+        CACert => "ca-cert.pem",
+    };
+
+    let dir = match sig_type {
```

Comment:
Would it make more sense to just implement `Display` for signature type?

Reply:
I think `Display` would be slightly misleading, since when you print (for example) `CACert` you would probably expect something like "CACert" or "root certificate", not the filename associated with the variant, which would be "ca-cert.pem" in this case.

Reply:
Ya, I was just thinking that `CACert` could display as `ca-cert`. Or more generally, the filename that corresponds to that particular enum should be information associated with the enum, rather than information associated with this helper function.

Reply:
It makes sense that `PemType` and `SigType` should own the information about themselves. Since the full path to a cert (ex. `certs/ec384/server-key.pem`) doesn't make sense to be owned by either one of them, I still want to keep the `get_cert_path()` function but just move the information about each enum to an impl for each enum (`get_filename()` for `PemType` and `get_dir_name()` for `SigType`).